set.mm

```diff
@@ -438886,11 +438884,11 @@ orthogonal vectors (i.e. whose inner product is 0) is the sum of the
 htmldef "_Se" as
     " <IMG SRC='_cse.gif' WIDTH=15 HEIGHT=19 ALT=' Se' TITLE='Se'> ";
   althtmldef "_Se" as " Se ";
-  latexdef "_Se" as "\mathrm{\;Se\;}";
+  latexdef "_Se" as "\mathrm{Se}";
```

Comment:
aren't these binary relations ?
in which case, \mathrel{\operatorname{Se}}

Reply:
It would be easier if you gave the label of the syntax definition.
If you're talking of https://us.metamath.org/mpeuni/wse.html and https://us.metamath.org/mpeuni/wfr.html, both are binary relations, so use `\mathrel`, since `R Se A` (and the other one), and not classes.